ur_controllers/CMakeLists.txt

```
@@ -46,4 +63,13 @@ install(FILES controller_plugins.xml
 
 ament_export_dependencies(${THIS_PACKAGE_INCLUDE_DEPENDS})
 
+ament_export_include_directories(
+        include
```

The old file used two spaces for indentation. let's keep this...
```suggestion
  include
```